flow/Makefile

```diff
@@ -98,20 +98,40 @@ default: finish
 #
 # ==============================================================================
 
-# Include design and platform configuration
-include $(DESIGN_CONFIG)
+#-------------------------------------------------------------------------------
+# Default target when invoking without specific target.
+.DEFAULT_GOAL := finish
 
-PUBLIC=nangate45 sky130hd sky130hs asap7
+#-------------------------------------------------------------------------------
+# Proper way to initiate SHELL for make
+SHELL          = /bin/bash
+.SHELLFLAGS    = -o pipefail -c
 
-ifneq ($(wildcard $(PLATFORM_BASE)/$(PLATFORM)),)
-  export PLATFORM_DIR = $(PLATFORM_BASE)/$(PLATFORM)
-else ifneq ($(findstring $(PLATFORM),$(PUBLIC)),)
-  export PLATFORM_DIR = ./platforms/$(PLATFORM)
-else ifneq ($(wildcard ../../$(PLATFORM)),)
-  export PLATFORM_DIR = ../../$(PLATFORM)
-else
-  $(error [ERROR][FLOW] Platform '$(PLATFORM)' not found)
-endif
```

Comment:
Why is this removed? It was being used as a security mechanism.

Reply:
Hmm... probably got lost in one of the edit.